sim/battle-actions.ts

```diff
@@ -301,6 +301,8 @@ export class BattleActions {
 			pokemon.side.zMoveUsed = true;
 		}
 
+		const oldActiveMove = this.battle.activeMove;
```

Comment:
(This seems to be the same as `move`, since that's what we set it from on line 247; at least, I can't see how it could have changed yet.)